src/dashboard/webserver/server.py

```diff
@@ -92,11 +92,15 @@ def ids_alerts():
 
 @app.route('/ids-priv/privacy-alerts/')
 def privacy_alerts():
+<<<<<<< HEAD
+    all_alerts = get_alerts('Privacy', g.db)
+=======
```

Comment:
Please update this to remove the <<<< HEAD and ======, that will cause thing Flask server to crash

Reply:
I swear I fixed all the merge issues. I'll get right on it.